plugins/package-managers/node/src/funTest/kotlin/PnpmFunTest.kt

```diff
@@ -26,6 +26,7 @@ import org.ossreviewtoolkit.analyzer.analyze
 import org.ossreviewtoolkit.analyzer.create
 import org.ossreviewtoolkit.analyzer.resolveSingleProject
 import org.ossreviewtoolkit.model.toYaml
+import org.ossreviewtoolkit.plugins.packagemanagers.node.pnpm.Pnpm
```

Comment:
This should move the test to a sub-package as well.

Reply:
Shall we really do this? We'll end up with one file per package.

Reply:
So far we have always aligned the package of the test with the package of the class under test. So yes, I think we should continue doing this, even if it results in having a single test in a package.